app.js

```diff
@@ -0,0 +1,19 @@
+import isYes from './isYes.js';
+
+const quizButton = document.getElementById('quizbutton');
+const total = document.getElementById('Score');
+
+quizbutton.addEventListener('click', () => {
+    console.log('buttonworks');
```

Comment:
Not too important but it's generally a good practice to remove any console logs before a code review.